### PR TITLE
I think I fixed it...fingers crossed

### PR DESCRIPTION
**Pull request by danitacodes**
Closes #

### What new changes did you make? Tick all applicable boxes
- [ x] Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### Describe the new changes you added.
Was working on getting the merge conflicts from the changelog

### CODE REVIEW

File: public/server/db/imu.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "definition": "I Miss U",
```
Comment (Njong392) [inline public/server/db/imu.json:2]: Seeing as the letter `U` in this case is a small abbreviation on its own, I'd rather we go with the alternative as the definition and leave it at `I Miss You`. What do you think?

**Review by Njong392 — CHANGES_REQUESTED** [review]

**Comment by Njong392** [comment]
Also, I'm curious to know how you got out of the situation. Loving the pull request title ahah

File: public/server/db/imu.json (hunk 1 above)
Comment (danitacodes) [reply public/server/db/imu.json:2]: We can definitely go with the alternative.

**Comment by danitacodes** [comment]
> I did what you suggested. I declined the 5 commits and then on my end, I had to approve the merge head conflict in the file. I am so happy that it worked =)

**Comment by Njong392** [comment]
Hello @danitacodes do you need any help updating this PR?

**Comment by danitacodes** [comment]
Hi, my apologies. I am going in to use the alternative and then submitting the pr again, correct?

**Comment by danitacodes** [comment]
Hi!
Thank you. I am learning so much.

I did it =)

**Review by Njong392 — APPROVED** [review]
Approved! 🎉 Thank you so much for the new abbreviations. YOU ROCK! 💣

**Pull request merged**